### PR TITLE
Some small cleanup - mainly removing Localpart from Conference.java

### PR DESCRIPTION
**Pull request by bbaldino**
Pass a`String` instead of a `Localpart` for the conference name.  This is something I came across when doing JVB API investigation, since we won't have a `Localpart` in the future.

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/Conference.java [hunk 1]
```diff
@@ -427,7 +428,21 @@ else if (logger.isDebugEnabled())
...
-        iq.setName(getName());
+        try
+        {
+            if (conferenceName == null)
+            {
+                iq.setName(null);
+            }
+            else
+            {
+                iq.setName(Localpart.from(conferenceName));
```
Comment (JonathanLennox) [inline src/main/java/org/jitsi/videobridge/Conference.java:439]: Conference names currently always come from localparts, so in practice this shouldn't contain any forbidden characters (throwing XmppStringprepException), right?

Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/Conference.java:439]: Yeah, the idea is that if they need to be put back into a Localpart, they must have come from a Localpart.

File: src/main/java/org/jitsi/videobridge/Conference.java [hunk 2]
```diff
@@ -1307,7 +1322,7 @@ public JSONObject getDebugState(boolean full, String endpointId)
     {
         JSONObject debugState = new JSONObject();
         debugState.put("id", id);
-        debugState.put("name", name == null ? null : name.toString());
+        debugState.put("name", conferenceName == null ? null : conferenceName);
```
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/Conference.java:1325]: That's equivalent to just `conferenceName`. Did you want the string "null"?

Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/Conference.java:1325]: Ha, oops.  I just wanted to recreate the old logic, which looks like would be just using `conferenceName`

Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/Conference.java]: Fixed, thanks.

File: src/main/java/org/jitsi/videobridge/Videobridge.java [hunk 3]
```diff
@@ -219,7 +219,7 @@ public Videobridge()
      */
     public @NotNull Conference createConference(Jid focus, Localpart name, String gid)
     {
-        return this.createConference(focus, name, /* enableLogging */ true, gid);
+        return this.createConference(focus, name == null ? null : name.toString(), /* enableLogging */ true, gid);
```
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/Videobridge.java:222]: Nit:  I think we can push this back one more level (`VideobridgeShim`) and simplify to `Objects.toString(name)`

**Pull request merged**